src/Extensions/Array.test.ts

```diff
@@ -0,0 +1,36 @@
+import { describe } from 'riteway'
+import './Array'
+
+describe('Array', async (should: any) => {
+  const { assert } = should('')
```

Comment:
I'm not sure whether this works as expected if we don't use the `should` from the immediate `describe`...

Reply:
ok, I don't know, but just in case I will move it